studio/data/cli/login.ts

```diff
@@ -0,0 +1,21 @@
+import { post } from 'data/fetchers'
+
+export async function createCliLoginSession(
+  sessionId: string,
+  publicKey: string,
+  tokenName?: string
+) {
+  if (!sessionId) {
+    throw new Error('sessionId is required')
+  }
+
+  const { data } = await post(`/platform/cli/login`, {
```

Comment:
needs an error handler?

const { data, error } = await post(...)

if (error) throw error

Reply:
could we make this an RQ mutation actually? that way we get some helpful handlers out of the box like isSuccess, isError etc etc

can refer to branch-create-mutation for example

Reply:
I was thinking about this, but because this is just a one-off call, that will instantly redirect us on either success or failure, it felt like an overkill.